### DISCUSSION TITLE
Internal GraphQL schema marks JsonFilter types as DateTime

### DISCUSSION THREAD

**Discussion by RobertCraigie** [prisma/prisma#9116] [Q&A] [answered]
While looking into adding support for `Json` types to https://github.com/RobertCraigie/prisma-client-py I looked through the generated GraphQL schema and noticed that the `JsonFilter` type takes values of `DateTime` type, I would've expected to see either `Json` or `String`. Should be noted that this happens with and without the preview feature enabled. 

Just curious as to whether or not this is intentional.

```graphql
input JsonFilter {
  equals: DateTime
  not: DateTime
}

input JsonWithAggregatesFilter {
  equals: DateTime
  not: DateTime
  _count: NestedIntFilter
  count: NestedIntFilter
  _min: NestedJsonFilter
  min: NestedJsonFilter
  _max: NestedJsonFilter
  max: NestedJsonFilter
}
```

**Comment by janpio** [comment, marked as answer]
Turned this into a bug issue at https://github.com/prisma/prisma/issues/9117

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
